docs/reference/aggregations/bucket/daterange-aggregation.asciidoc

```diff
@@ -111,3 +111,35 @@ Zone:: 'Z' outputs offset without a colon, 'ZZ' outputs the offset with a colon,
 Zone names:: Time zone names ('z') cannot be parsed.
 
 Any characters in the pattern that are not in the ranges of ['a'..'z'] and ['A'..'Z'] will be treated as quoted text. For instance, characters like ':', '.', ' ', '#' and '?' will appear in the resulting time text even they are not embraced within single quotes.
+
+[[time-zones]]
+==== Time zone in date range aggregations
+
+Dates can be converted from another time zone to UTC by specifying the `time_zone` parameter.
+
+Time zones may either be specified as an ISO 8601 UTC offset (e.g. +01:00 or -08:00) or as one of
+the the http://joda-time.sourceforge.net/timezones.html[time zone ids] from the TZ database.
```

Comment:
The list at http://www.joda.org/joda-time/timezones.html tends to be very out of date. It is not regularly updated.

Joda-Time sticks to the IANA data, so there should not be additional zones.